types/simulation/account.go

```diff
@@ -14,10 +14,11 @@ import (
 // eventually more useful data can be placed in here.
 // (e.g. number of coins)
 type Account struct {
-	PrivKey cryptotypes.PrivKey
-	PubKey  cryptotypes.PubKey
-	Address sdk.AccAddress
-	ConsKey cryptotypes.PrivKey
+	PrivKey       cryptotypes.PrivKey
+	PubKey        cryptotypes.PubKey
+	Address       sdk.AccAddress
+	ConsKey       cryptotypes.PrivKey
+	AddressBech32 string
```

Comment:
We should store the bytes and use the address codec wherever we need this.

Reply:
The encoding had some significant performance impact. The sim accounts are not modified during the whole sims run for a seed. Therefore it made sense for me to do the encoding once in the beginning and then use this field instead of re-encoding. This field is only used by sims code and not in the app or modules. WDYT?